Point_set_3/include/CGAL/Point_set_3.h

```diff
@@ -474,6 +473,38 @@ class Point_set_3
     return out;
   }
 
+  /*! 
+    \brief Convenience method to copy a point with all its properties
+    from another point set.
+
+    In the case where two point sets have the same properties, this
+    method allows the user to easily copy one point (along with the
+    values of all its properties) from one point set to another.
+
+    \param other Point set to which the point to copy belongs
+    \param idx Index of the point to copy in `other`
+
+    \warning This point set and `other` must have the exact same
+    properties, with the exact same names and types in the exact same
+    order.
```

Comment:
Is the order that important? And what is the order? The `add` order? I guess internally this is a map: string -> map so if the names and the same the order should be the same too.

Reply:
This comment remains to be answered.

Reply:
Sorry, I must have missed it.

Yes, the order is important (the `add` order indeed), and no, internally this is not a map but a vector of abstract property maps. The function iterates at the same time on the properties of both point sets and copies the values from one to the other.